src/transformers/angular-fe-schema-transformer.ts

```diff
@@ -9,6 +9,9 @@ export const AngularFormEngineSchemaTransformer: FormSchemaTransformer = {
         });
       }
     });
+    if(form?.meta?.programs) {
+      handleProgramMetaTags(form);
+    }
```

Comment:
- Fix formatting issues here. 
- Can the form object be null/undefined?

Reply:
The meta part will be except for people migrating from afe

Reply:
Yes for the meta object it may be undefined but not the form object so:

```
form.meta?.programs
```

Can you also fix the formatting issues?

Reply:
Done